packet/ldap.py

```diff
@@ -1,9 +1,12 @@
+"""
+Helper functions for working with the csh_ldap library
+"""
+
 from functools import lru_cache
 
 from packet import _ldap
 
 
-@lru_cache(maxsize=1024)
```

Comment:
I'm unsure of any benefit to not caching ldap functions. Ldap doesn't change very often, but it is incredibly slow

Reply:
I went for the strategy of only caching things that are used at page load. This is only used by the CLI so the cache just added an extra layer of complexity that didn't provide any benefit.